go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/lima-vm/lima
 
-go 1.21
+go 1.22.0
```

Comment:
Could we use `go 1.22`?

Reply:
No, the syntax was changed to require the patch version too.

Reply:
Should we use the latest available patch version?
```suggestion
go 1.22.6
```

Reply:
I don't think it is necessary.
It is user's responsibility to update Go.

Reply:
Can we merge?

Reply:
`go 1.22` seems to work with the current toolchains

Reply:
CI is passing with `go 1.22`, so let me merge this